freespace2/freespace.cpp

```diff
@@ -2935,8 +2935,13 @@ void apply_view_shake(matrix *eye_orient)
 	tangles.h = 0.0f;
 	tangles.b = 0.0f;
 
-	// do shakes that only affect the HUD (unless disabled by cmdline in singleplayer)
-	if (Viewer_obj == Player_obj && (!Cmdline_no_screenshake || Game_mode & GM_MULTIPLAYER)) {
+	bool do_hud_shudder = (Viewer_obj == Player_obj);
+
+	if ((Viewer_mode & VM_CHASE) && !Apply_shudder_to_chase_view)
```

Comment:
Shouldn't this be ``&& Apply_shudder_to_chase_view`` instead? 
At least by name, the flag suggests that it explicitly enables shudder in chase mode, not the other way around

Reply:
Yes it should. I remember testing this one pretty thoroughly.. I must have been cleaning up some code and that snuck in. Built and tested just now to double check.